pyproject.toml

```diff
@@ -65,42 +65,16 @@ exclude = ["tests*"]
 [tool.setuptools_scm]
 write_to = "src/zino/version.py"
 
-[tool.black]
-line-length = 120
-# Exclude files even when passed directly as argument (for MegaLinter)
-force-exclude = '''
-(
-  /(
-      \.eggs         # exclude a few common directories in the
-    | \.git          # root of the project
-    | \.hg
-    | \.mypy_cache
-    | \.nox
-    | \.tox
-    | \.venv
-    | \.ve
-    | _build
-    | buck-out
-    | build
-    | dist
-    | src/zino/snmp/mibdumps
-    | docs
-  )
-)
-'''
```

Comment:
Does ruff implicitly ignore these?

Reply:
Yes, it automatically ignores everything in `.gitignore` (https://docs.astral.sh/ruff/settings/#respect-gitignore) and others are also automatically ignored: https://docs.astral.sh/ruff/settings/#exclude

Reply:
@johannaengland how about `src/zino/snmp/mibdumps` ?

Reply:
That is already ignored: https://github.com/Uninett/zino/blob/31f8b41f4ac16180ced21d5db7fe9cb6db2ec7e7/pyproject.toml#L95-L97

But I just noticed that I should use `extend_exclude` there, not `exclude`